changelogs/fragments/570_postgresql_info.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - postgresql_info - fix SQL syntax issue (https://github.com/ansible-collections/community.postgresql/issues/570)
```

```suggestion
  - postgresql_info - fix SQL syntax issue (https://github.com/ansible-collections/community.postgresql/issues/570).
```

for changelog entry consistency